eslint.config.mjs

```diff
@@ -0,0 +1,107 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+import typescriptPlugin from 'typescript-eslint';
+
+const files = {
+  build: [
+    '*.js',
+    '*.mjs'
+  ],
+  client: [
+    'src/client/**/*.ts',
+    'src/client/**/*.js'
+  ],
+  test: [
+    'src/test/*/*.ts'
+  ],
+  node_test: [
```

Comment:
Quite unexpected 🐍

Reply:
What is unexpected there? Tests simply contain parts that are node, and parts that are _client_.

Reply:
(Happy to follow-up if there is stuff we can improve).

Reply:
snake_case, sorry for unclear comment

Reply:
Ok :snake: Now I got your emoji.